### ISSUE TITLE
💡 Setting up PlantUML for Mac (Using Homebrew)

### ISSUE THREAD

**Issue by sbhbenjamin** [nus-cs2103-AY2122S2/forum#232]
Hello, I thought I'd share this in case someone runs into the same problem while setting up PlantUML on a mac.  
When you are setting up PlantUML following the [SE-EDU Tutorial](https://se-education.org/guides/tutorials/plantUml.html), you will first have to install GraphWiz:

> 1. Install Graphviz. Graphviz is a dependency that PlantUML requires to generate more advanced diagrams. Head over to the [downloads page](https://www.graphviz.org/download/) on the official Graphviz website and follow instructions to install Graphviz.
> 2. Install the PlantUML integration plugin in IntelliJ IDEA.
>     1. Go to `Settings` > `Plugins` > `Marketplace` and install the plugin `PlantUML integration`.
>     2. Then go to `Settings` > `Languages & Frameworks` > `PlantUML` or search for PlantUML. **Configure the path to the `dot` executable. This executable can be found in the `/bin` directory where you installed GraphViz earlier.**

For step 2.ii, I was unable to find the dot file. After some peering around, the `dot` file was stored at `/opt/homebrew/Cellar/graphviz/3.0.0/bin/dot` for homebrew users. You can key this path in into the `Graphiz dot executable` field in step 2.ii.
  
Hope this helps in some way!

**Comment by aweijun** [comment]
Thanks! This was super helpful! 😄

**Comment by jbkim1999** [comment]
Thank you! I was struggling with this issue as well, this approach certainly works on my M1 mac too!

**Comment by punnyhuimin** [comment]
I'm using a Mac, but this unfortunately didn't work for me. 😢 I found a StackOverflow thread on how to find the dot executable file and now it's working. 
https://stackoverflow.com/questions/54966989/where-do-i-add-graphvizs-executable-on-a-mac 

TLDR: 
1. Download GraphViz via homebrew
```
brew install graphviz
```
2. Now, you need to get the directory of the dot executable. I tried to find it in my finder but couldn't do so as the folder is not visible to me. What you need to do is to run 
```
which dot
```
This returns the directory in which the dot executable file is at. 
<img width="277" alt="Screenshot 2022-04-06 at 3 30 48 PM" src="https://user-images.githubusercontent.com/61443625/161919938-e1386256-48d8-413e-9818-d15523c19ec7.png">

3. Lastly, just copy the directory and paste it into the GraphViz dot executable section.
<img width="987" alt="Screenshot 2022-04-06 at 3 32 12 PM" src="https://user-images.githubusercontent.com/61443625/161920221-7682e101-d706-4f3f-92d5-2b6a262802f2.png">

4. Click "Ok" and restart Intellij to see your diagrams 💯
